src/components/dialogs/pick-entity-or-conversion.vue

```diff
@@ -194,24 +227,25 @@ export default class PickEntityOrConversionDialog extends CommonMixin {
     return output
   }
 
-  get width (): string {
-    if (this.showSocietiesInfo || this.isConversion) {
+  get maxWidth (): string {
```

Comment:
Needed for double-wide table layout.

See screenshot attached to comment above for example.